src/MainWindow.ui

```diff
@@ -3028,6 +3028,43 @@ Hint: keep your mouse positioned over an option to get more details.</string>
               </item>
              </layout>
             </item>
+            <item>
+             <layout class="QHBoxLayout" name="horizontalLayout_46">
+              <item>
+               <widget class="QLabel" name="labelHIBPCheck">
+                <property name="text">
+                 <string>HIBP check: Enabled</string>
```

Comment:
Same here HIBP -> Have I Been Pwnd

Also it would be nice to have a clickable link below that goes to the https://haveibeenpwned.com website to let user see what this service is.